taxonomies/categories.txt

```diff
@@ -1417,41 +1417,304 @@ agribalyse_proxy_food_name:en:Beer, dark
 agribalyse_proxy_food_name:fr:Bière brune
 
 <en:Beers
-en:India Pale Ale (IPA), IPA ales, India pale ale, India pale ales, IPA beers
-es:Cerveza IPA, IPA
+en:Packaged beers, Beers in a package, Packed beers
+lt:Supakuotas alus, Alus pakuotėje, Alus dėžėje
+
+################################# ALES #################################
+
+<en:Beers
+en:Ales, Ale
+lt:Elis
+wikidata:en:Q208385
+
+<en:Ales
+en:Pale ales, Pale ale
+
+############ IPA ############
+
+<en:Pale ales
 ca:Cervesa IPA, IPA
+en:India Pale Ale (IPA), IPA ales, India pale ale, India pale ales
+es:Cerveza IPA, IPA
+fr:Bières IPA, Bieres IPA, Bière ipa
 xx:India Pale Ale (IPA)
 wikidata:en:Q1543214
 
 <en:India Pale Ale (IPA)
-en:Milkshake IPA, Milkshake India Pale Ale
-wikidata:en:Q103351397
+xx:Amber IPA
 
 <en:India Pale Ale (IPA)
-en:New England IPA, NEIPA ales, New England India Pale ale
-lt:New England IPA alus, NEIPA alus
+<en:American beers
+xx:American IPA
+lt:Amerikietiškas IPA
+
+<xx:American IPA
+xx:Cornish IPA
+
+<xx:American IPA
+<en:Strong beers
+xx:Double IPA, Imperial IPA, Extra IPA, DIPA
+
+<xx:American IPA
+xx:East Coast IPA, East IPA
+
+<xx:American IPA
+xx:Hurricane IPA, Tornado IPA
+
+<xx:American IPA
+xx:New England IPA, NEIPA, Hazy IPA, Juicy IPA, Unfiltered IPA
```

Comment:
the hazy / juicy / unfiltered IPA should not be added as synonyms I think, there could be "unfiltered IPAs" from lots of other places

Reply:
On a related note, New England IPA should probably not be under American IPA, it's a description of a style, not an origin.

e.g. this page lists lots of NEIPA brewed in France: https://unepetitemousse.fr/blog/neipa-new-england-ipa/